RadClass/H0.py

```diff
@@ -55,12 +68,18 @@ def run_gross(self, data, timestamp):
             self.x2 = data
             n = self.x1 + self.x2
             p = 0.5
-            pval = stats.binom_test(self.x1, n, p, alternative='two-sided')
-
-            # only save instances with rejected null hypothesesf
+            # scipy.stats.binomtest will fail if n (# of trials)
+            # is less than 1 (possible for low count-rate integration times)
+            if int(n) < 1:
+                pval = 1.0
+            else:
+                pval = stats.binomtest(int(self.x1), int(n), p,
+                                       alternative='two-sided').pvalue
```

Comment:
Note the `.pvalue`. This is required since `scipy.stats.binomtest` now returns a `BinomTestResults` instance, which contains `k`, `n`, `pval`, etc.